src/main/java/com/spotinst/sdkjava/model/SpotInstOceanGKELaunchSpecClient.java

```diff
@@ -0,0 +1,118 @@
+public class SpotInstOceanGKELaunchSpecClient {
+
+    private static final Logger LOGGER =
+            LoggerFactory.getLogger(SpotinstElastigroupClientGcp.class);
+    //region Members
+    private               String authToken;
+    private               String account;
+
+    public SpotInstOceanGKELaunchSpecClient(String authToken, String account) {
+
+        this.authToken = authToken;
+        this.account = account;
+    }
+
+    public OceanGKELaunchSpec createOceanGKELaunchSpec(OceanGKECreateLaunchSpecRequest oceanGKECreateLaunchSpecRequest){
+
+        OceanGKELaunchSpec retVal;
+        OceanGKELaunchSpec oceanGKECreateLaunchSpecRes = oceanGKECreateLaunchSpecRequest.getOceanGKECreateLaunchSpecRes();
+        SpotinstRepoManager                 managerInstance     = SpotinstRepoManager.getInstance();
+        ISpotinstOceanGKECreateLaunchSpecRepo         repoGcp             = managerInstance.getSpotinstOceanGKECreateLaunchSpecRepoGcp();
+        RepoGenericResponse<OceanGKELaunchSpec> creationResponse    =
+                repoGcp.create(oceanGKECreateLaunchSpecRes, authToken, account);
+
+        if (creationResponse.isRequestSucceed()) {
+            retVal = creationResponse.getValue();
+        }
+        else {
+            List<HttpError> httpExceptions = creationResponse.getHttpExceptions();
+            HttpError       httpException  = httpExceptions.get(0);
+            LOGGER.error(
+                    String.format("Error encountered while attempting to create a launch spec. Code: %s. Message: %s.",
+                                  httpException.getCode(), httpException.getMessage()));
+            throw new SpotinstHttpException(httpException.getMessage());
+        }
+
+        return retVal;
+
+    }
+
+    public OceanGKELaunchSpec getALaunchSpec(String oceanId){
+        OceanGKELaunchSpec retVal = null;
+        SpotinstRepoManager                 managerInstance     = SpotinstRepoManager.getInstance();
+        ISpotinstOceanGKECreateLaunchSpecRepo         repoGcp             = managerInstance.getSpotinstOceanGKECreateLaunchSpecRepoGcp();
+        RepoGenericResponse<OceanGKELaunchSpec> creationResponse    =
+                repoGcp.get(oceanId,authToken,account);
+        if (creationResponse.isRequestSucceed()) {
+            retVal = creationResponse.getValue();
+        }
+        else {
+            List<HttpError> httpExceptions = creationResponse.getHttpExceptions();
+            HttpError       httpException  = httpExceptions.get(0);
+            LOGGER.error(
+                    String.format("Error encountered while attempting to fetch a  launch spec. Code: %s. Message: %s.",
+                                  httpException.getCode(), httpException.getMessage()));
+            throw new SpotinstHttpException(httpException.getMessage());
+        }
+        return retVal;
+    }
+
+    public Boolean updateOceanGKELaunchSpec(OceanGKECreateLaunchSpecRequest updateLaunchSpecRequest,String launchSpecId){
```

Comment:
Should simply be named as updateLaunchSpec for consistency with other functions.